docs/docs/hypermedia-protocol/document-blocks.md

```diff
@@ -117,10 +117,12 @@ One of:
 
 - [`Heading`](#heading-block)
 - [`Paragraph`](#paragraph-block)
+- [`Code Block`](#code-block)
+- [`Math Block`](#math-block)
 
-Text blocks should have a value for the `text` field, and `annotations` are used to further describe the text content.
+Text blocks should have a value for the `text` field, and optionally `annotations` are used to further enrich the text content.
 
-### File Blocks
+### Media Blocks
```

Comment:
Embed blocks don't have a file so I was using "File Blocks" to describe blocks which reference IPFS files, which is different from "Media Blocks"